cartography/data/indexes.cypher

```diff
@@ -254,10 +254,10 @@ CREATE INDEX ON :AzureVirtualMachineAvailableSize(id);
 CREATE INDEX ON :AzureVirtualMachineScaleSet(id);
 CREATE INDEX ON :AzureVirtualMachineScaleSetExtension(id);
 CREATE INDEX ON :AzureKeyVault(id);
-CREATE INDEX ON :AzureUser(name);
```

Comment:
did we change the field name from name to id?

Reply:
yes